apps/src/code-studio/pd/application/teacher/ChooseYourProgram.jsx

```diff
@@ -47,6 +47,17 @@ const CourseHoursLabeledNumberInput = props => {
   );
 };
 
+const TeachingPlansNote = () => {
```

Comment:
nit: I have a hard time intuiting what this text says by this name. Could it be something like `ScholarshipEligibilityWarning`?

Reply:
I also realize you diid *not* decide on this name!

Reply:
Will do! Good idea, thank you!